pkg/controller/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -279,7 +290,12 @@ func (r *OAMApplicationReconciler) Reconcile(req reconcile.Request) (result reco
 	log.Debug("Successfully rendered components", "workloads", len(workloads))
 	r.record.Event(ac, event.Normal(reasonRenderComponents, "Successfully rendered components", "workloads", strconv.Itoa(len(workloads))))
 
-	if err := r.workloads.Apply(ctx, ac.Status.Workloads, workloads, resource.MustBeControllableBy(ac.GetUID())); err != nil {
+	applyOpts := []resource.ApplyOption{resource.MustBeControllableBy(ac.GetUID())}
+	if r.applyOnceOnly {
+		log.Debug("ApplyOnceOnly is enabled")
```

Comment:
```suggestion
		log.Info("ApplyOnceOnly is enabled")
```

This message deserves info level

Reply:
please move this log at start time, not for all CR reconcile